boot/unlocked_state.go

```diff
@@ -9,6 +9,7 @@ import (
 )
 
 // states for partition state
+// TODO: transform those into enums
```

Comment:
I don't think this make sense because of the potential version skew between kernel and user space, int enums are much more fragile from that POV. Unless the plan is to use enums in memory and strings on disk?

Reply:
I removed it.